generated/client/client.go

```diff
@@ -251,6 +258,50 @@ func (c Client) CreateBook(ctx context.Context, i *models.CreateBookInput) (*mod
 
 	default:
 		return nil, models.DefaultInternalError{Msg: "Unknown response"}
+	}
+}
+
+func (c Client) HealthCheck(ctx context.Context, i *models.HealthCheckInput) error {
```

Comment:
looks like `i` is unused, which would be slightly confusing from the perspective of someone consuming this as a client. I think this is an edge case, though.